platforms/registry/src/config/database 2.ts

```
@@ -0,0 +1,18 @@
+import { DataSource } from "typeorm"
+import { Vault } from "../entities/Vault"
+import * as dotenv from "dotenv"
+import { join } from "path"
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)